src/test/java/org/opensearch/sdk/TransportCommunicationIT.java

```diff
@@ -35,7 +35,9 @@ public class TransportCommunicationIT extends OpenSearchIntegTestCase {
     private final int port = 7777;
     private final String host = "127.0.0.1";
     private volatile String clientResult;
+    private ExtensionsRunner extensionsRunner;
     private NettyTransport nettyTransport = new NettyTransport();
+    private NettyTransport nettyTransport1 = new NettyTransport(extensionsRunner);
```

Comment:
You'll need to initialize the `extensionsRunner` object 2 lines up, otherwise this will fail.  And you should be deleting the no-arg constructor call so you don't need a copy with 1 appended.